core/src/main/scala/org/apache/spark/MapOutputTracker.scala

```diff
@@ -725,11 +726,13 @@ private[spark] class MapOutputTrackerMaster(
   // Make sure that we aren't going to exceed the max RPC message size by making sure
   // we use broadcast to send large map output statuses.
   if (minSizeForBroadcast > maxRpcMessageSize) {
-    val msg = s"${SHUFFLE_MAPOUTPUT_MIN_SIZE_FOR_BROADCAST.key} ($minSizeForBroadcast bytes) " +
-      s"must be <= spark.rpc.message.maxSize ($maxRpcMessageSize bytes) to prevent sending an " +
-      "rpc message that is too large."
+    val msg = log"${MDC(CONFIG, SHUFFLE_MAPOUTPUT_MIN_SIZE_FOR_BROADCAST.key)} " +
+      log"(${MDC(MIN_SIZE, minSizeForBroadcast.toString)} bytes) " +
+      log"must be <= spark.rpc.message.maxSize (${MDC(MAX_SIZE, maxRpcMessageSize.toString)} " +
+      log"bytes) to prevent sending an rpc message that is too large."
+
     logError(msg)
-    throw new IllegalArgumentException(msg)
+    throw new IllegalArgumentException(msg.message)
```

Comment:
This seems to be touched as the example. Shall we revert this?

Reply:
Sure, reverted